drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md

```diff
@@ -0,0 +1,47 @@
+# 🌶 Spice up your server-side rendered apps with Stimulus and Turbolinks
+
+These days, SPA's (Single Page Applications) are an undeniable trend on the web. However, I personally think there are many use cases where a 'traditional' server-side rendered application still makes sense, for example for CRUD-style / admin applications or in general when your application doesn't require highly interactive features.
+
+For use cases like these, it's generally more convenient and faster to use server side rendering. Still, also these kinds of applications can benefit from small bits of interactivity like a `copy to clipboard` feature, selecting multiple checkboxes in a table at once etc.
```

Comment:
`these kind of applications`, kinds (= plural) would indicate multiple kinds of applications, sounds way more than previously mentioned